requirements-dev.txt

```diff
@@ -7,12 +7,7 @@ pytest-flask
 mongomock<=3.12
 responses
 
-# packaging
-wheel>=0.23
-twine
-
 # utils
-black
```

Comment:
Pre-commit creates is own environment and downloads black so it is not needed in the requirements file

Reply:
I use black but not pre-commit since I think it is a bit cumbersome, since it is an option to not use pre-commit I think it is good to have black in the dev requirements

Reply:
Ok if you both want to keep it I'll let it stay